packages/icons/general/TriangleWarning.tsx

```diff
@@ -0,0 +1,13 @@
+import { createIcon } from '../utils'
+
+export const TriangleWarning = createIcon(
+    'TriangleWarningIcon',
+    <g fill="none">
+        <path opacity=".2" d="m10 .84 10 17.32H0L10 .84Z" fill="#FF5F5F" />
```

Comment:
`fill={MaskColorVar.redMain.alpha(1)}`

Reply:
`MaskColorVar` cann't be used in the package